doc/running.md

```diff
@@ -34,24 +64,55 @@ helm repo update
 
 ## <a name="env"></a>Environment variables
 
-At a mininum, you need to set the following environment variable:
+The simplest solution is to ask for a copy of a working `.env` file from one of
+the other developers with the envars set for development.
```

Comment:
`envars` is a new one on me!

```suggestion
the other developers with the env vars set for development.
```
is maybe a more recognizable abbreviation? ;-)